docs/cli/usage/iam.md

```diff
@@ -64,6 +64,8 @@ The Amplify CLI requires the below IAM policies for performing actions across al
                 "appsync:ListResolvers",
                 "appsync:DeleteType",
                 "appsync:GetType",
+                "appsync:TagResource",
```

Comment:
I believe this action is already present in the doc here - https://docs.amplify.aws/cli/usage/iam
The untag one is not present. @ammarkarachi do we need it?

Reply:
Yes we do, let's do a review for other resources too and update it.

Reply:
It looks like TagResource was added after I made this PR 6 weeks ago. Is there anything I need to do as part of this PR thread, or are you all going to take over from here?